core/src/test/java/com/yahoo/imapnio/async/data/StoreResultTest.java

```diff
@@ -0,0 +1,54 @@
+package com.yahoo.imapnio.async.data;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.sun.mail.iap.ProtocolException;
+import com.sun.mail.imap.protocol.FetchResponse;
+import com.sun.mail.imap.protocol.IMAPResponse;
+import com.sun.mail.imap.protocol.MODSEQ;
+
+/**
+ * Unit test for {@code StoreResult}.
+ */
+public class StoreResultTest {
+    /**
+     * Tests StoreResult constructor and getters.
+     */
+    @Test
+    public void testStorehResult() throws IOException, ProtocolException {
+        final IMAPResponse imapResponse = new IMAPResponse("* 1 FETCH (UID 4 MODSEQ (12121231000))");
```

Comment:
This is handled in ImapResponseMapperTest.java line 1188. FetchResult is also tested in the similar way. Otherwise, it will fail branch test coverage.